src/Spago/Command/Build.purs

```diff
@@ -153,17 +151,20 @@ run opts = do
         let reportTest = pedanticPackages || (fromMaybe false $ p.package.test >>= _.pedantic_packages)
         Alternative.guard (reportSrc || reportTest)
         pure $ Tuple p { reportSrc, reportTest }
-    unless (Array.null pedanticPkgs || opts.depsOnly) do
+    if Array.null pedanticPkgs || opts.depsOnly then do
```

Comment:
well this clearly wasn't it

Reply:
I guess it should be the other way around 😄 
```
if Array.null pedanticPkgs || opts.depsOnly then do
  pure true
else do
```

Reply:
yeah that's what I meant by the comment but I still have 2 spago docs tests failing

Reply:
Linux CI is happy!